tests/drucker_prager_derivatives_2d.cc

```diff
@@ -5,13 +5,20 @@
 #include <aspect/simulator_access.h>
 #include <aspect/newton.h>
 #include <aspect/utilities.h>
+#include <aspect/parameters.h>
+
+#include <deal.II/base/exceptions.h>
+#include <memory>
+#include <functional>
 
 #include <iostream>
 
 template<int dim>
-int f()
+void f(const aspect::SimulatorAccess<dim> &simulator_access,
+       aspect::Assemblers::Manager<dim> &)
 {
-  std::cout << std::endl << "Test with dimension " << dim << std::endl;
+
+  std::cout << std::endl << "Testing DruckerPrager derivatives against analytical derivatives " << std::endl;
```

Comment:
could you check the above sentence? Shouldn't it be: `Testing DruckerPrager derivatives against finite difference derivatives`? Same for the other test.

Reply:
@MFraters - yes, the wording you used above is correct. Thanks for catching that!